examples/Remarketing/AddExpressionRuleUserList.cs

```diff
@@ -0,0 +1,158 @@
+            userListRuleItemGroupInfo.RuleItems.Add(rule2);
+
+            UserListRuleInfo userListRuleInfo = new UserListRuleInfo();
+            userListRuleInfo.RuleItemGroups.Add(userListRuleItemGroupInfo);
+
+            // Creates an ExpressionRuleUserListInfo object, or a boolean rule that defines this
+            // user list. The default rule_type for a UserListRuleInfo object is OR of ANDs
+            // (disjunctive normal form). That is, rule items will be ANDed together within rule
+            // item groups and the groups themselves will be ORed together.
+            ExpressionRuleUserListInfo expressionRuleUserListInfo = new ExpressionRuleUserListInfo
+            {
+                Rule = userListRuleInfo
+            };
+
+            // Defines a representation of a user list that is generated by a rule.
+            RuleBasedUserListInfo ruleBasedUserListInfo = new RuleBasedUserListInfo
+            {
+                // Optional: To include past users in the user list, set the prepopulation_status to
+                // REQUESTED.
+                PrepopulationStatus = UserListPrepopulationStatus.Requested,
+                ExpressionRuleUserList = expressionRuleUserListInfo
+            };
+
+            // Creates a new user list.
+            UserList userList = new UserList
+            {
+                Name = "All visitors to example.com/section1 AND example.com/section2 " +
+                       $"#{ExampleUtilities.GetRandomString()}",
+                Description = "Visitors of both example.com/section1 AND example.com/section2",
+                MembershipStatus = UserListMembershipStatus.Open,
+                MembershipLifeSpan = 365L,
+                RuleBasedUserList = ruleBasedUserListInfo
+            };
+
+            // Creates the operation.
+            UserListOperation operation = new UserListOperation()
+            {
+                Create = userList
+            };
+
+            try
+            {
+                // Adds the user list.
+                MutateUserListsResponse response = userListServiceClient.MutateUserLists
+                    (customerId.ToString(), new[] {operation});
+
+                // Displays the results.
+                Console.WriteLine("Created new user list with resource name: " +
+                                  $"'{response.Results.First().ResourceName}'.");
+            }
+            catch (GoogleAdsException e)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {e.Message}");
+                Console.WriteLine($"Failure: {e.Failure}");
+                Console.WriteLine($"Request ID: {e.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Creates and returns a UserListRuleItemInfo object targeting a visit to a specified URL.
+        /// </summary>
+        /// <param name="url">The URL at which the rule will be targeted.</param>
```

Comment:
I don't think this makes sense. Maybe something like "The URL at which the rule will target users."?

Reply:
Fixed.